gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthorizationService.js

```diff
@@ -344,6 +353,46 @@ var fluid = fluid || require("infusion");
     // ==== End of addAuthorization()
 
     /**
+     * A common utility function shared by checkAuthDecision() and doAdd()
+     * This function retrieves and returns an authorization decision id that associates with the given gpii token and the client.
```

Comment:
"that associates with" isn't perfectly clear - could we describe the relationship between the decision id, the token and the client more precisely?